websites/S/Steam/metadata.json

```diff
@@ -27,4 +34,4 @@
 		"community",
 		"gaming"
 	]
-}
\ No newline at end of file
+}
```

Comment:
Thibk about implementing multilang?
metadata.json examples:
https://github.com/PreMiD/Presences/blob/fd0123a557d92e87cba5617d3c505a66472a2de6/websites/M/MEE6/metadata.json#L25-L29

presence.ts examples:
https://github.com/PreMiD/Presences/blob/fd0123a557d92e87cba5617d3c505a66472a2de6/websites/M/MEE6/presence.ts#L5-L16
https://github.com/PreMiD/Presences/blob/fd0123a557d92e87cba5617d3c505a66472a2de6/websites/M/MEE6/presence.ts#L33-L34
https://github.com/PreMiD/Presences/blob/fd0123a557d92e87cba5617d3c505a66472a2de6/websites/M/MEE6/presence.ts#L41-L52

Reply:
idk, I leave it up to the original creator of the presence